compilers/compiler.js

```diff
@@ -31,4 +34,16 @@ exports.compile = function (load, opts, plugin) {
     source: output.code,
     sourceMap: sourceMap
   });
-};
\ No newline at end of file
+};
+
+exports.transformAMDDependencies = function (load) {
+  var options = {
+    plugins: [[require('babel-plugin-transform-amd-system-dependencies').default]]
+  };
+
+  var output = babel.transform(load.source, options);
+
+  return {
+    amdDeps: output.metadata.amdDeps
+  };
+}
```

Comment:
This should probably still use the default Babel options.

Also we should add `ast: true` so we can return and cache the ast and not need to parse twice.

Then we can probably just return the Babel output directly and call this a more general `exports.compileAst` which returns `metadata` and `ast`.